.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+jobs:
+  build:
+    branches:
+      only:
+        - zfs-0.7-release
+  build:
+    docker:
+      - image: ubuntu:14.04
+    environment:
+      - ZFS_BUILD_TAGS=0
+      - CODECOV_TOKEN=04a04811-63ec-4800-b1d6-f741dc7e079a
+    working_directory: github.com/openebs/zfs
+    steps:
+      - checkout
+      - run:
+          command: |
+            # install dependencies
+            apt-get update
+            apt-get install bc
+            sudo apt-get install git -y
+            sudo apt-get install -y software-properties-common
+            sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
+            sudo apt-get update
+            sudo apt-get install gcc-snapshot -y
+            sudo apt-get update
+            sudo apt-get install gcc-6 g++-6 -y
+            sudo apt-get update
+            sudo apt-get install -y build-essential autoconf libtool gawk alien fakeroot linux-headers-$(uname -r) libaio-dev
+            sudo apt-get install -y zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev libssl-dev
+            sudo apt-get install -y lcov libjemalloc-dev
+
+            # packages for tests
+            sudo apt-get install -y parted lsscsi ksh attr acl nfs-kernel-server fio
+            sudo apt-get install -y libgtest-dev cmake
+
+            # packages for debugging
+            sudo apt-get install -y gdb
+
+            # use gcc-6 by default
+            sudo unlink /usr/bin/gcc && sudo ln -s /usr/bin/gcc-6 /usr/bin/gcc
+            sudo unlink /usr/bin/g++ && sudo ln -s /usr/bin/g++-6 /usr/bin/g++
+            pushd .
+            cd /usr/src/gtest
+            sudo cmake CMakeLists.txt
+            sudo make -j4
+            sudo cp *.a /usr/lib
+            popd
+            cd ..
+
+            # we need fio repo to build zfs replica fio engine
+            git clone https://github.com/axboe/fio
+            cd fio
+
+            # TODO replace commit hash by version tag in future
+            git checkout 2e4ef4fbd69eb6d4c07f2f362463e3f3df2e808c
```

Comment:
there was a recent change to .travis.yml which should be done here as well:
```
-    # TODO: replace commit hash by version tag in future
-    - git checkout 2e4ef4fbd69eb6d4c07f2f362463e3f3df2e808c
+    - git checkout fio-3.7
```

Reply:
Addressed